internal/graph/check.go

```diff
@@ -626,9 +634,6 @@ func (c *LocalChecker) checkRewrite(
 	case *openfgapb.Userset_This:
 		return c.checkDirect(ctx, req)
 	case *openfgapb.Userset_ComputedUserset:
-		ctx, span := tracer.Start(ctx, "checkComputedUserset")
-		defer span.End()
```

Comment:
this span is not needed. `c.dispatch` will create the right span when it calls `ResolveCheck`